great-idea/css/index.css

```diff
@@ -49,18 +130,245 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+body {
+	line-height: 1;
+}
+ol,
+ul {
+	list-style: none;
+}
+blockquote,
+q {
+	quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+	content: "";
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	height: 100%;
+	font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+* {
+	box-sizing: border-box;
+	padding: 10;
+	margin: 0;
+	border: 1px solid grey;
+}
+
+header nav {
+	display: flex;
+	justify-content: space-evenly;
+	color: lightgray;
+	margin: 35px 60px 60px;
+	text-align: center;
+}
+nav a {
+	text-decoration: none;
+	color: grey;
+	font-size: 13px;
+	letter-spacing: 1.5px;
+	margin-left: 10px;
+	margin-right: 10px;
+}
+nav a:hover {
+	/*styles are applied when a user hovers over this element*/
+	color: #4682b4;
+	text-decoration: overline;
+}
+
+.top-page {
+	display: flex;
+	flex-grow: 2;
+	margin-bottom: 3%;
+	padding: 5px;
+	justify-content: center;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+.box {
+	display: flex;
+	flex-direction: column;
+	width: 20%;
+	justify-content: flex-end;
+	padding: 5px;
+	text-align: center;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+.main-heading {
+	display: flex;
+	flex-direction: column;
+}
+button {
+	background-color: white !important;
+	color: black;
+	width: 35%;
+	height: 10%;
+	padding: 0.5px;
+	font-size: 10px;
+	margin-top: 2%;
+}
+
+h1 {
+	font-size: 50px;
+	color: black;
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.headerimg {
+	display: flex;
+	height: 35%;
+	width: auto;
+}
+
+.above {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	width: 100%;
+	height: 100px;
```

Comment:
setting the height here is not necessary. could put space around paragraph div's by just using margin and padding